frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts

```diff
@@ -231,16 +228,25 @@ export const gitPage = {
         );
     }
   },
-  verifyValidatedMessage: (gitUrl = 'https://github.com/sclorg/nodejs-ex.git') => {
```

Comment:
This is the default param value

Reply:
But there is another function which enters the URL (`gitPage.enterGitUrl)

Using another default URL here could result in really strange things since the code now calculates the component name based on the URL. (Earlier it was just logged.)

That's the reason I ensured that this function is always be called with an URL.

Reply:
Makes sense